### ISSUE TITLE
Speed up C++ builds via mounting ccache directory as a docker volume

### ISSUE THREAD

**Issue by kszucs** [ursa-labs/ursabot#136]
The plan here is to use mount a volume for each builder under `~/.ccache` similarly like the [JS builder does](https://github.com/ursa-labs/ursabot/blob/master/ursabot/builders.py#L688). This would spare ~3 minutes out of the current ~6 minutes builds.

```python
class ArrowCppTest(DockerBuilder):
    volumes = [
        lambda builder: f'{slugify(builder.name)}:/root/.ccache:rw'
    ]
```

@pitrou is it safe enough to use the same ccache directory between revisions, including that the dependencies are also evolving over time.

**Comment by pitrou** [comment]
> @pitrou is it safe enough to use the same ccache directory between revisions

Probably? However, you probably need a different volume for each builder (I'm not sure concurrent access would work).

**Comment by kszucs** [comment]
One volume per builder is the original plan, just not sure about ccache "consistency over speed" capabilities.

**Comment by kszucs** [comment]
resolved by #135
